src/client/components/Checker.tsx

```diff
@@ -28,11 +33,31 @@ export const Checker: FC<CheckerProps> = ({ config }) => {
     }
   }
 
+  const renderDisplay = (display: checker.Display, i: number) => {
+    const values = display.targets.map((output) => variables[output])
+    switch (display.type) {
+      case 'TEXT':
+        return <TextDisplay key={i} content={values[0]} {...display} />
+      case 'BUTTON':
+        return (
+          <ButtonDisplay
+            key={i}
+            buttonText={values[0]}
+            buttonUrl={values[1]}
+            {...display}
+          />
+        )
+    }
+  }
+
   const onSubmit = (inputVariables: Record<string, string | number>) => {
     const computedVariables = operations.reduce(
       evaluateOperation,
       inputVariables
     )
+    constants.forEach(
```

Comment:
Some of the operations might reference constants as well

Reply:
Good catch, have made constants as a hidden form input in https://github.com/opengovsg/checkfirst/pull/22/commits/c25ca4fce1a5cc5071fae5540f82b311d5a8ab8f